frontend/src/components/MeetingRoom/BackgroundEffectsLayout/BackgroundEffectsLayout.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { ReactElement, useCallback, useEffect, useState } from 'react';
+import { Box, Button, Typography } from '@mui/material';
+import usePublisherContext from '../../../hooks/usePublisherContext';
+import RightPanelTitle from '../RightPanel/RightPanelTitle';
+import EffectOptionButtons from '../../BackgroundEffects/EffectOptionButtons/EffectOptionButtons';
+import BackgroundGallery from '../../BackgroundEffects/BackgroundGallery/BackgroundGallery';
+import BackgroundVideoContainer from '../../BackgroundEffects/BackgroundVideoContainer';
+import useBackgroundPublisherContext from '../../../hooks/useBackgroundPublisherContext';
+import { DEFAULT_SELECTABLE_OPTION_WIDTH } from '../../../utils/constants';
+import AddBackgroundEffect from '../../BackgroundEffects/AddBackgroundEffect/AddBackgroundEffect';
+import getInitialBackgroundReplacement from '../../../utils/videoFilterUtils';
+
+export type BackgroundEffectsLayoutProps = {
+  handleClose: () => void;
+  isOpen: boolean;
+};
+
+/**
+ * BackgroundEffectsLayout Component
+ *
+ * This component manages the UI for background effects (cancel background and blurs) in a room.
+ * @param {BackgroundEffectsLayoutProps} props - The props for the component.
+ *   @property {boolean} isOpen - Whether the background effects panel is open.
+ *   @property {Function} handleClose - Function to close the panel.
+ * @returns {ReactElement} The background effects panel component.
+ */
+const BackgroundEffectsLayout = ({
+  handleClose,
+  isOpen,
+}: BackgroundEffectsLayoutProps): ReactElement | false => {
+  const [backgroundSelected, setBackgroundSelected] = useState('none');
```

Comment:
could we specify the type for the `useState` as `<string>`? Ideally, I'd set it to just an empty string, too, and update the check for an empty string instead of checking for `none`